src/AngularPublic.js

```diff
@@ -262,7 +262,7 @@ function publishExternalAPI(angular) {
         $window: $WindowProvider,
         $$rAF: $$RAFProvider,
         $$jqLite: $$jqLiteProvider,
-        $$HashMap: $$HashMapProvider,
+        $$Map: $$MapProvider,
```

Comment:
This is a private service, so no breaking change (theoretically). It is used by `ngAnimate` (which doesn't have access to the closure).